grails-app/i18n/messages.properties

```diff
@@ -1300,7 +1300,7 @@ kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCo
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.districtId.modal.button.ok = Yes, delete all the votes and supports
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.districtId.modal.button.cancel = Ouch, no. Let it as is.
 kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.cause.label = Cause
-kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.label = Type
+kuorum.web.commands.payment.participatoryBudget.DistrictProposalChooseDistrictCommand.backerType.label = BackerType
```

Comment:
This text should be 2 words, isn't it? -> Backer Type => Fix it on POE as well

Reply:
It has been changed on POE